OpenRA.Game/Settings.cs

```diff
@@ -20,23 +20,61 @@ namespace OpenRA
 {
 	public enum MouseScrollType { Disabled, Standard, Inverted }
 
+	public class LaunchSettings
```

Comment:
Please, no.  These are configured as launch-only flags instead of settings because we should not be persisting them across launches!